include/translated-post.php

```diff
@@ -82,9 +82,9 @@ protected function register_language_taxonomy(): void {
 			(array) $this->get_translated_object_types(),
 			array(
 				'labels' => array(
-					'name'          => __( 'Languages', 'polylang' ),
-					'singular_name' => __( 'Language', 'polylang' ),
-					'all_items'     => __( 'All languages', 'polylang' ),
+					'name'          => 'Languages',
+					'singular_name' => 'Language',
+					'all_items'     => 'All languages',
```

Comment:
See my global [comment](https://github.com/polylang/polylang/pull/1550#issuecomment-2387814900)
I don't see what it really imply to remove translated labels here.

Reply:
As far as I can tell, these labels are not used anywhere. I think we could even remove them entirely. @Chouby doesn't recall any use of them neither.